app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -272,11 +270,11 @@ private function createUser($record)
             $userData[$key] = $record->{$key};
         }
     
-        if (!empty($record->email_subscription_status)) {
+        if (isset($record->email_subscription_status)) {
```

Comment:
Won't `isset` be `true` even if `$record->email_subscription_status` is `false`?

Reply:
Yup! Our `$record->email_subscription_status` has cast the user's email opt-in preferences to a boolean here, so my understanding is that we'd want to update Northstar with `true` or `false` for these new users. We use this to determine whether to send the Activate Account welcome email.

Reply:
OH wow okay I reversed the red and green lines in my head and that's why I was confused! I'm on board with this!